provider/repodata/data/json/currency/essential@1/ar.json

```diff
@@ -0,0 +1,1215 @@
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 221,
+      "narrow_place_holder_index": 65535
+    },
+    "XPF": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 222,
+      "narrow_place_holder_index": 65535
+    },
+    "XXX": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 223,
+      "narrow_place_holder_index": 65535
+    },
+    "YER": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 224,
+      "narrow_place_holder_index": 65535
+    },
+    "ZAR": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 225,
+      "narrow_place_holder_index": 226
+    },
+    "ZMW": {
+      "short_pattern_standard": true,
+      "narrow_pattern_standard": true,
+      "short_place_holder_index": 227,
+      "narrow_place_holder_index": 228
+    }
+  },
+  "standard": "‏#,##0.00 ¤;‏-#,##0.00 ¤",
+  "standard_alpha_next_to_number": "",
+  "place_holders": [
+    "د.إ.‏",
+    "AFN",
+    "؋",
+    "ALL",
+    "AMD",
+    "֏",
+    "ANG",
+    "AOA",
+    "Kz",
+    "ARS",
+    "AR$",
+    "AU$",
+    "AWG",
+    "AZN",
+    "₼",
+    "BAM",
+    "KM",
+    "BBD",
+    "BB$",
+    "BDT",
+    "৳",
+    "BGN",
+    "د.ب.‏",
+    "BIF",
+    "BMD",
+    "BM$",
+    "BND",
+    "BN$",
+    "BOB",
+    "Bs",
+    "R$",
+    "BSD",
+    "BS$",
+    "BTN",
+    "BWP",
+    "P",
+    "BYN",
+    "р.",
+    "BYR",
+    "BZD",
+    "BZ$",
+    "CA$",
+    "CDF",
+    "CHF",
+    "CLP",
+    "CL$",
+    "CNH",
+    "CN¥",
+    "COP",
+    "CO$",
+    "CRC",
+    "₡",
+    "CUC",
+    "$",
+    "CUP",
+    "CU$",
+    "CVE",
+    "CZK",
+    "Kč",
+    "DJF",
+    "DKK",
+    "kr",
+    "DOP",
+    "DO$",
+    "د.ج.‏",
+    "ج.م.‏",
+    "E£",
+    "ERN",
+    "₧",
+    "ETB",
+    "€",
+    "FJD",
+    "FJ$",
+    "FKP",
+    "£",
+    "UK£",
+    "GEL",
+    "₾",
+    "GHS",
+    "GH₵",
+    "GIP",
+    "GMD",
+    "GNF",
+    "FG",
+    "GTQ",
+    "Q",
+    "GYD",
+    "GY$",
+    "HK$",
+    "HNL",
+    "L",
+    "HRK",
+    "kn",
+    "HTG",
+    "HUF",
+    "Ft",
+    "IDR",
+    "Rp",
+    "₪",
+    "₹",
+    "د.ع.‏",
+    "ر.إ.",
+    "ISK",
+    "JMD",
+    "JM$",
+    "د.أ.‏",
+    "JP¥",
+    "KES",
+    "KGS",
+    "KHR",
+    "៛",
+    "KMF",
+    "CF",
+    "KPW",
+    "₩",
+    "د.ك.‏",
+    "KYD",
+    "KY$",
+    "KZT",
+    "₸",
+    "LAK",
+    "₭",
+    "ل.ل.‏",
+    "L£",
+    "LKR",
+    "Rs",
+    "LRD",
+    "$LR",
+    "Lt",
+    "Ls",
+    "د.ل.‏",
+    "د.م.‏",
+    "MDL",
+    "MGA",
+    "Ar",
+    "MKD",
+    "MMK",
+    "K",
+    "MNT",
+    "₮",
+    "MOP",
+    "أ.م.",
+    "MUR",
+    "MVR",
+    "MWK",
+    "MX$",
+    "MYR",
+    "RM",
+    "MZN",
+    "NAD",
+    "NGN",
+    "₦",
+    "NIO",
+    "C$",
+    "NOK",
```

Comment:
Do you actually need to repeat all the ISO codes here?

Reply:
Good point, those `iso` code is a place holder, if we removed them, how could we differentiate between the missing placeholders, and the placeholders that equals to their iso code?

Reply:
Don't we always fall back to ISO code?